src/main/java/fi/metatavu/famifarm/rest/V1RESTService.java

```diff
@@ -2027,7 +2027,8 @@ private Response createHarvestEvent(fi.metatavu.famifarm.persistence.model.Produ
 
     HarvestEventType harvestType = eventData.getType();
     HarvestEvent event = harvestEventController.createHarvestEvent(product, startTime, endTime, harvestType,
-        productionLine, sowingTime, additionalInformation, amount, gutterHoleCount, baskets,  creatorId);
+      productionLine, sowingTime, additionalInformation, amount, gutterHoleCount, eventData.getCuttingHeight(),
```

Comment:
This is fine, although might be cleaner to have the named variable declared above rather than in line here- as the others are.